<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DADialogDataFrameFillna</class>
 <widget class="QDialog" name="DADialogDataFrameFillna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fillnaGroupBox">
     <property name="title">
      <string>Fillna</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="valueFilledCheckBox">
          <property name="text">
           <string>Filled By Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="valueSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="padCheckBox">
        <property name="text">
         <string>pad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ffillCheckBox">
        <property name="text">
         <string>ffill</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="backfillCheckBox">
        <property name="text">
         <string>backfill</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bfillCheckBox">
        <property name="text">
         <string>bfill</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>backfillCheckBox</zorder>
     <zorder>padCheckBox</zorder>
     <zorder>horizontalLayoutWidget_2</zorder>
     <zorder>ffillCheckBox</zorder>
     <zorder>bfillCheckBox</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/DAGui/icon/accept.svg</normaloff>:/DAGui/icon/accept.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../icon.qrc">
         <normaloff>:/DAGui/icon/cancel.svg</normaloff>:/DAGui/icon/cancel.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
